CHANGELOG.md

```diff
@@ -35,6 +38,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Removed the `WithMaxAttempts` and `WithBackoff` options from the `go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp` package.
   The retry logic of the package has been updated to match the `otlptracegrpc` package and accordingly a `WithRetry` option is added that should be used instead. (#2095)
 - Removed metrics test package `go.opentelemetry.io/otel/sdk/export/metric/metrictest`. (#2105)
+- Removed `DroppedAttributeCount` field from `otel/trace.Link` struct. (#2118)
```

Comment:
I think the convention would be to write `"go.opentelemetry.io/otel/trace".Link` instead of  `otel/trace.Link`.

Reply:
Oh, I just saw this and previous comment, but I guess the PR got merged to include it into `v1.0.0-RC2` released today.

If needed, I can normalize these to fit the convention in a separate PR, just let me know!